core/src/main/java/dev/hypera/scaffolding/Scaffolding.java

```diff
@@ -26,20 +26,17 @@
 import dev.hypera.scaffolding.schematic.Schematic;
 import dev.hypera.scaffolding.schematic.readers.MCEditSchematicReader;
 import dev.hypera.scaffolding.schematic.readers.SpongeSchematicReader;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.concurrent.CompletableFuture;
 import org.jetbrains.annotations.NotNull;
 import org.jglrxavpok.hephaistos.nbt.CompressedProcesser;
 import org.jglrxavpok.hephaistos.nbt.NBTCompound;
 import org.jglrxavpok.hephaistos.nbt.NBTException;
 import org.jglrxavpok.hephaistos.nbt.NBTReader;
 
+import java.io.*;
```

Comment:
`*` imports should not be used, `java.` imports should not be separate from the other imports.